### ISSUE TITLE
Error with react v0.20 - this._easing is undefined

### ISSUE THREAD

**Issue by labithiotis** [oblador/react-native-image-progress#11]
When I updated to v0.20 (from v0.18) I get the following error using the image loader and progress components.

![screen shot 2016-03-05 at 12 48 59](https://cloud.githubusercontent.com/assets/784306/13547596/22b185e2-e2d1-11e5-85e9-b93465da2f88.png)

```javascript

<ImageLoader
	indicator={Progress.CircleSnail}
	indicatorProps={{
		size: this.props.indicatorSize || 60,
		color: colour.blue
	}}
	threshold={100}
	{...this.props}
/>
```

**Comment by oblador** [comment]
This is a bug solved in [react-native-progress 2.2.1](https://github.com/oblador/react-native-progress/releases/tag/v2.2.1).

**Comment by labithiotis** [comment]
Whoops should have check that :(
